sdk/go/asymmetric_decrypt.go

```diff
@@ -0,0 +1,89 @@
+package ehsm
+
+import (
+	"encoding/json"
+	"fmt"
+	ioutil "io/ioutil"
+
+	"github.com/iancoleman/orderedmap"
+)
+
+type AsymmetricDecryptResponse struct {
+	Code    int    `json:"code"`
+	Message string `json:"message"`
+	Result  struct {
+		Plaintext string `json:"plaintext"`
+	} `json:"result"`
+}
+
+/*
+Description:
+Decrypt an arbitrary set of bytes using the CMK.(only support asymmetric types).
+Input:
+keyid    		-- The keyid of the asymmetric cmk.
+ciphertext 		-- The data of the ciphertext in BASE64 string.
+padding mode	--padding_mode is necessary when keyspec is RSA.
+
+	-EH_RSA_PKCS1
+	-EH_RSA_PKCS1_PSS
+	-EH_PAD_NONE
+
+Output:
+plaintext -- Plaint data after decrypt and stored in BASE64 string.
+*/
+func (c *Client) AsymmetricDecrypt(keyid, ciphertext, padding_mode string) (string, error) {
+	payload := orderedmap.New()
+	if keyid != "" {
+		payload.Set("keyid", keyid)
+	} else {
+		return "", fmt.Errorf("keyid is empty.")
+	}
+	if padding_mode != "" {
+		payload.Set("padding_mode", padding_mode)
+	} else {
+		return "", fmt.Errorf("padding_mode is empty.")
+	}
+	if ciphertext != "" {
+		payload.Set("ciphertext", ciphertext)
+	} else {
+		return "", fmt.Errorf("ciphertext is empty.")
+	}
+
+	params := initParams(c.appid, payload, c.apikey)
```

Comment:
for this initParams can be hide the params of appid and apikey?

not sure whether it can call c.appid and c.apikey dirrectly inside the initParams function.

Reply:
fix